### ISSUE TITLE
multilightpack results in errors/warnings

### ISSUE THREAD

**Issue by Sp0rks** [hyperion-project/hyperion.ng#505]
Hi,
 I have 3 light packs connected to a Rasberry PI 2.0 (running Stretch) and after spending longer than I care to admit trying to compile Hyperion.NG I finally got it working but am experiencing issues where it throws various errors and warnings regarding the packs, doesn't always detect the 3 of them and even if it does, throws errors regardless and a majority of the time only 1 (if I'm lucky 2) of the packs will actually light up.

This is what it gives me when running in verbose

```
pi@RASPBERRY:~ $ sudo hyperiond -v hyperion.json
[hyperiond MAIN] <INFO> Selected configuration file: /root/.hyperion/config/hyperion.json
[hyperiond LOGGER] <WARNING> Logger settings overridden by command line argument
[hyperiond Core] <INFO> ColorAdjustment 'default' => [0; 29]
[hyperiond ComponentRegister] <INFO> Json/Proto forwarder: off
[hyperiond LedDevice] <INFO> LedDevice 'multilightpack' configured.
[hyperiond LedDevice] <INFO> USB context initialized in multi Lightpack device
[hyperiond LedDevice] <INFO> Found a lightpack device. Retrieving serial...
[hyperiond LedDevice] <ERROR> Lightpack device found with serial 953303139393512180B1
[hyperiond LedDevice] <INFO> Found a lightpack device. Retrieving serial...
[hyperiond LedDevice] <ERROR> Lightpack device found with serial 95330313939351D0D102
[hyperiond LedDevice] <INFO> Found a lightpack device. Retrieving serial...
[hyperiond LedDevice] <ERROR> Lightpack device found with serial 9523030323135121C0F0
[hyperiond LedDevice] <INFO> Found a lightpack device. Retrieving more information...
[hyperiond LedDevice] <INFO> Lightpack device successfully opened
[hyperiond LedDevice] <INFO> Found a lightpack device. Retrieving more information...
[hyperiond LedDevice] <INFO> Found a lightpack device. Retrieving more information...
[hyperiond LedDevice] <INFO> Lightpack device successfully opened
[hyperiond LedDevice] <INFO> Found a lightpack device. Retrieving more information...
[hyperiond LedDevice] <INFO> Found a lightpack device. Retrieving more information...
[hyperiond LedDevice] <INFO> Found a lightpack device. Retrieving more information...
[hyperiond LedDevice] <INFO> Lightpack device successfully opened
[hyperiond LedDevice] <INFO> 3 Lightpack devices were found
[hyperiond Core] <INFO> Creating linear smoothing
[hyperiond Smoothing] <INFO> smoothing cfg 0: interval: 4501504 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 1: pause
[hyperiond ComponentRegister] <INFO> Smoothing: activated
[hyperiond ComponentRegister] <INFO> LED device: activated
[hyperiond Smoothing] <INFO> smoothing cfg 2: interval: 18627536 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 3: interval: 18649254 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 4: interval: 18652966 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 5: interval: 18606554 ms, settlingTime: 50 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 6: interval: 18608150 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 7: interval: 18609042 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 8: interval: 18582534 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 9: interval: 18608286 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 10: interval: 18609250 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 11: interval: 18610190 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 12: interval: 18580898 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 13: interval: 18612770 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 14: interval: 18581294 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 15: interval: 18584174 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 16: interval: 18585594 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 17: interval: 18581038 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 18: interval: 18587986 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 19: interval: 18631490 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 20: interval: 18586902 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 21: interval: 18588002 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 22: interval: 18633782 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 23: interval: 18634630 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 24: interval: 18634886 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 25: interval: 18637442 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 26: interval: 18637594 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 27: interval: 18634930 ms, settlingTime: 50 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 28: interval: 18636366 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 29: interval: 18498574 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 30: interval: 18498174 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 31: interval: 18499490 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 32: interval: 18605546 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 33: interval: 18605462 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 34: interval: 18605374 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 35: interval: 18604934 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 36: interval: 18604942 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond Smoothing] <INFO> smoothing cfg 37: interval: 18604878 ms, settlingTime: 40 ms, updateDelay: 0 frames
[hyperiond EFFECTENGINE] <INFO> 35 effects loaded from directory :/effects/
[hyperiond EFFECTENGINE] <INFO> 19 effect schemas loaded from directory :/effects/schema/
[hyperiond EFFECTENGINE] <INFO> 0 effects loaded from directory /root/.hyperion/custom-effects
[hyperiond MAIN] <INFO> Hyperion initialized
[hyperiond MAIN] <INFO> Json server created and started on port 19444
QObject::connect: No such slot ProtoServer::setVideoMode(VideoMode)
[hyperiond MAIN] <INFO> Proto server created and started on port 19445
[hyperiond ComponentRegister] <INFO> Boblight server: off
[hyperiond Core] <INFO> Register new input source named 'UDPLISTENER' for priority channel '200'
[hyperiond ComponentRegister] <INFO> UDP listener: off
[hyperiond ComponentRegister] <INFO> Blackborder detector: activated
[hyperiond V4L2:auto] <INFO> Signal threshold set to: {12, 12, 12}
[hyperiond V4L2:auto] <INFO> Signal detection area set to: 0.250000,0.250000 x 0.750000,0.750000
[hyperiond ComponentRegister] <INFO> V4L capture device: off
[hyperiond MAIN] <INFO> set screen capture device to 'dispmanx'
[hyperiond ComponentRegister] <INFO> Blackborder detector: activated
[hyperiond DISPMANXGRABBER] <INFO> Display opened with resolution: 1280x720
[hyperiond Core] <INFO> Register new input source named 'Dispmanx' for priority channel '250'
[hyperiond MAIN] <INFO> DISPMANX frame grabber created and started
[hyperiond ComponentRegister] <INFO> Framegrabber: activated
[hyperiond MAIN] <INFO> Hyperion started
[hyperiond EFFECTENGINE] <INFO> run effect Rainbow swirl fast on channel 0
[hyperiond Core] <INFO> Register new input source named 'Rainbow swirl fast' for priority channel '0'
[hyperiond MAIN] <INFO> Inital foreground effect 'Rainbow swirl fast' started
[hyperiond EFFECTENGINE] <INFO> run effect Warm mood blobs on channel 254
[hyperiond Core] <INFO> Register new input source named 'Warm mood blobs' for priority channel '254'
[hyperiond MAIN] <INFO> Inital background effect 'Warm mood blobs' started
[hyperiond WEBSERVER] <INFO> started on port 8090 name 'Hyperion WebConfig'
[hyperiond LedDevice] <WARNING> Unable to write data to Lightpack device: no more led data available
[hyperiond LedDevice] <WARNING> Unable to write data to Lightpack device: no more led data available
[hyperiond LedDevice] <WARNING> Unable to write data to Lightpack device: no more led data available
[hyperiond LedDevice] <WARNING> Unable to write data to Lightpack device: no more led data available
[hyperiond LedDevice] <WARNING> Unable to write data to Lightpack device: no more led data available
[hyperiond LedDevice] <WARNING> Unable to write data to Lightpack device: no more led data available
[hyperiond LedDevice] <WARNING> Unable to write data to Lightpack device: no more led data available
[hyperiond LedDevice] <WARNING> Unable to write data to Lightpack device: no more led data available
[hyperiond LedDevice] <WARNING> Unable to write data to Lightpack device: no more led data available
[hyperiond LedDevice] <WARNING> Unable to write data to Lightpack device: no more led data available
[hyperiond LedDevice] <WARNING> Unable to write data to Lightpack device: no more led data available
[hyperiond LedDevice] <WARNING> Unable to write data to Lightpack device: no more led data available
[hyperiond LedDevice] <WARNING> Unable to write data to Lightpack device: no more led data available
[hyperiond LedDevice] <WARNING> Unable to write data to Lightpack device: no more led data available
```

and it'll spam that WARNING about no more led data until I kill it.

I completely understand this is still in ALPHA but thought I'd play around with it

**Comment by penfold42** [comment]
The error re: serial numbers isn’t an error - it should be marked as info.

How many leds do you have configured ?

**Comment by Sp0rks** [comment]
I have 30 LEDs configured

I have the three packs, the usb capture device and flirc connected to a powered usb hub which is then connected to the pi

this was one of the good examples where it detected all three packs (even though only one actually turned on), it quite often gives a LIBUSB_ERROR_PIPE or something along those lines when it runs through its detection routine and wont return all the serial numbers, the current stable version of hyperion does this too occasionally but restarting the service (quite often multiple times) usually fixes it

If I configure it to run just one pack, it'll actually work and wont spam that warning regarding no more led data available

**Comment by Lord-Grey** [comment]
@Sp0rks It is already a while since you opened this issue....
Would you mind checking, if your problems get resolved by #855?

**Comment by Lord-Grey** [comment]
Closed, as no feedback by @Sp0rks
